src/tests/ui/planner/workitem-quickadd.ts

```diff
@@ -43,4 +43,13 @@ export class WorkItemQuickAdd extends ui.BaseElement {
       return filtered;
     }, []);
   }
+  
+  async addAndOpenWorkItem({ title, description = '', type = 'feature' }: WorkItem) {
```

Comment:
Can we use the `workitem` type as the param type?

Reply:
i think for the type it's better to pass it as string param...see addAndOpenWorkItem() ...and also there are 8 workitem types and using union types is not a good idea..WDYT?